Assets/Scripts/Audio/FMODMissionComplete.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FMODMissionComplete : MonoBehaviour
+{
+    [FMODUnity.ParamRef]
+    public string parameter;
+    public bool isComplete = false;
+
+    private FMOD.Studio.PARAMETER_DESCRIPTION parameterDescription;
+    private Interactable interactable;
+
+    void Start()
+    {
+        interactable = Utils.GetRequiredComponent<Interactable>(this);
+        interactable.OnInteractEnd += missionComplete;
+        FMODUnity.RuntimeManager.StudioSystem.getParameterDescriptionByName(parameter, out parameterDescription);
+    }
+
+    void OnTriggerEnter(Collider other)
+    {
+        if (!isComplete)
+        {
+            FMODUnity.RuntimeManager.StudioSystem.setParameterByName(parameter, 1f);
+            isComplete = true;
+        }
+    }
+
+    private void missionComplete(Interactable source)
```

Comment:
Usual function naming convention for functions is starting with a capital letter, also I'd recommend maybe the name `OnMissionComplete` instead?

Reply:
Done!